src/components/views/MapView.js

```diff
@@ -67,10 +67,14 @@ class MapView extends Component {
             return array.push([element.latitude, element.longitude]);
         });
         timeline.map(elem => {
-            let points = gpsData.filter(e => {
-                return (elem.distance - e.distance) < 0.5 && (elem.distance - e.distance) >= -0.5
-            });
-            let temp = points[points.length -1];
+            let points;
+            for(var i = 0.05; i <= 0.5; i += 0.05){
+                 points = gpsData.filter(e => {
+                    return (elem.distance - e.distance) < i && (elem.distance - e.distance) >= -i
+                 });
+                 if(points.length > 0) {break};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1116)